### PR TITLE
feat(testing): use chai instead of @bundled-es-modules/chai

### PR DESCRIPTION
**Pull request by daKmoR**
- use chai directly
- allows to easily use every chai plugin (no need for es module)
- add chai-dom to default chai plugins
- add example for sinon-chai

- [ ] Update Readme
- [ ] Decide if feat release is good enough

fixes https://github.com/open-wc/open-wc/issues/729

### CODE REVIEW

File: packages/testing/index.js [hunk 1]
```diff
@@ -18,4 +19,5 @@ export {
   elementUpdated,
 } from '@open-wc/testing-helpers/index.js';
 
-export { chai, expect, should, assert } from '@bundled-es-modules/chai';
+// @ts-ignore
+export const { expect, should, assert } = window.chai;
```
Comment (LarsDenBakker) [inline packages/testing/index.js]: does this work well with type checking? or is that solved in the other PR?

File: packages/testing/test/chaiA11yAxe.test.js [hunk 2]
```diff
@@ -3,76 +3,62 @@ import { fixture, expect, assert, html } from '../index.js';
 describe('chaiA11yAxe', () => {
   describe('Expect', () => {
     it('passes axe accessible tests', async () => {
-      const el = await fixture(
-        html`
-          <button>some light dom</button>
-        `,
-      );
+      const el = await fixture(html`
```
Comment (LarsDenBakker) [inline packages/testing/test/chaiA11yAxe.test.js]: isn't this the prettier formatting style?

File: packages/testing/index.js (hunk 1 above)
Comment (daKmoR) [reply packages/testing/index.js]: yeah I wanna rebase the typing branch on this one and see if it "works/helps" before merging this

File: packages/testing/test/chaiA11yAxe.test.js (hunk 2 above)
Comment (daKmoR) [reply packages/testing/test/chaiA11yAxe.test.js]: prettier allows for both styles 👍  you will need to roughly format it like this so it keeps the auto format - imho 2 lines less is worth the effort

File: packages/testing/index.js (hunk 1 above)
Comment (LarsDenBakker) [reply packages/testing/index.js]: Ok, LGTM otherwise so just merge whenever.

**Pull request merged**